test/e2e-go/cli/algod/expect/algod_expect_test.go

```diff
@@ -19,10 +19,12 @@ import (
 	"testing"
 
 	"github.com/algorand/go-algorand/test/framework/fixtures"
+	"github.com/algorand/go-algorand/test/partitiontest"
 )
 
 // TestAlgodWithExpect Process all expect script files with suffix Test.exp within the test/e2e-go/cli/algod/expect directory
 func TestAlgodWithExpect(t *testing.T) {
+	partitiontest.PartitionTest(t)
```

Comment:
This should be be called here because it is already partitioned in `et.Run()` with better partitioning. 
Partitioning here will result in double partitioning, leaving many tests out.

Reply:
this is on purpose, to test our logic, will remove before merging.